### ISSUE TITLE
Hyper-parameter for each model and dataset

### ISSUE THREAD

**Issue by Chanrom** [XiaofengZhu/uRank_uMart#7]
Thanks for sharing the code. Would you kindly share the hyper-parameter for each model or dataset? For example, your param.json file. Thanks.

**Comment by XiaofengZhu** [comment]
Thank you for your inquiry for your interest in our work. I will try to locate and pack the json files (there will be20 files =  5 models * 4 datasets) in a couple of weeks. In the meantime, you can try the parameter range addressed in the paper. For example, "The two hidden fully-connected layers in uRank, uBoost, andurBoost have dimensions [100, 50] for the OSHUMED dataset, di-mensions [100, 100] for the MQ2007 dataset, and dimensions [200,200] for the MSLR datasets" and "Mart-based models use 200-1000trees/learners with 31-255 leaves per tree and 255-10,000 bins.." In addition, please feel free to add additional layers for the MLP layers and adjust the learning rate, etc.

**Comment by XiaofengZhu** [comment]
The hyper-parameters that matter most are the mlp_sizes and learning_rate. I list the values I used for the four datasets as follows.

OSHUMED
mlp_sizes: [100, 50]
learning_rate: 1e-4

MQ2007
mlp_sizes: [100, 100]
learning_rate: 1e-4

MSLR-10K
mlp_sizes: [200, 200]
learning_rate: 1e-5


MSLR-30K
mlp_sizes: [200, 200]
learning_rate: 1e-5

Please feel free to add additional layers to the mlp_sizes, eg., [200, 200, 200].

I hope this helps.

**Comment by Chanrom** [comment]
Thanks for sharing the details. I will first try different mlp_sizes and learning_rates.
